source/features/highlight-collaborators-and-own-conversations.tsx

```diff
@@ -42,7 +42,7 @@ void features.add(import.meta.url, {
 		pageDetect.isRepoIssueOrPRList,
 	],
 	exclude: [
-		() => select.exists('.blankslate'),
+		pageDetect.isBlank,
```

Comment:
Regarding `isBlank`, I want to mention that `.blankslate` always exists on some pages (e.g. https://github.com/refined-github/refined-github/compare/main...esbuild-2), maybe we need to review usage of it to fix false positives

Reply:
Refer to https://github.com/refined-github/refined-github/issues/5222#issuecomment-1168304082 before making large changes